src/vec.rs

```diff
@@ -58,18 +58,22 @@ assert_eq!(
 			// here go all the other filters
 			LabelMatch { name: "bar".to_string(),      op: Eq, value: "baz".to_string(), },
 		],
-		range: None, offset: None,
+		range: None, resolution: None, offset: None,
 	}))
 );
 # }
 ```
 */
+
 #[derive(Debug, PartialEq)]
 pub struct Vector {
 	/// Set of label filters
 	pub labels: Vec<LabelMatch>,
 	/// Range for range vectors, in seconds, e.g. `Some(300)` for `[5m]`
 	pub range: Option<usize>,
+        /// Resolution for subqueries, defaults to the global evaluation_interval (1m). Same
+        /// time-semantics as range or offset.
+        pub resolution: Option<usize>,
```

Comment:
- I think it would be helpful to give here an example of parsed value, similar to `range` and `offset` fields
- This introduces space-indented lines into otherwise tab-indented file.

Reply:
Thanks, I'll fix that up.